code/game/machinery/autolathe.dm

```diff
@@ -344,7 +352,8 @@
 		for(var/i=1, i<=multiplier, i++)
 			var/obj/item/new_item = new being_built.build_path(A)
 			new_item.autolathe_crafted(src)
-
+			d_disk.used_charges += 1
+			d_disk.check_charges()
```

Comment:
probably will work. maybe. I should probably test this but idk
```suggestion
			if(being_built in d_disk.blueprints)
				d_disk.used_charges += 1
				d_disk.check_charges()
```

Reply:
we have a perfectly good playerbase to test on